libraries/helpers.rb

```diff
@@ -328,3 +328,27 @@ def check_deprecated_properties
     end
   end
 end
+
+#
+# Chef Stack Helpers
+#
+def prefix
+  (platform_family?('windows') ? 'C:/Chef/' : '/etc/chef/')
+end
```

Comment:
I concur. I just figured for this first merge to leave the resources unaltered. Next PR we can start refactoring inline with the rest of the resources. You ok with that approach?